src/pages/scheduler/components/CourseCard.tsx

```diff
@@ -68,14 +88,29 @@ export function CourseCard({
           </Flex>
         </Flex>
         <Flex direction="row" alignItems="center" justifyContent="space-between" w="100%">
-          <VStack alignItems="start" spacing="0" py="2">
-            <Text fontSize="lg" fontWeight="bold">
-              {subject} {code}
-            </Text>
-            <Text fontSize="sm" fontWeight="normal">
-              <Skeleton isLoaded={!loading}>{data?.title ?? ''}</Skeleton>
-            </Text>
-          </VStack>
+          <Flex grow={1}>
+            <VStack
+              alignItems="start"
+              spacing="0"
+              py="2"
+              tabIndex={0}
+              as={Link}
+              to={`/calendar/${term}/${subject}?pid=${pid}`}
+            >
+              <Text
+                fontSize="lg"
+                fontWeight="bold"
+                _hover={{
```

Comment:
If you add the `_hover` prop to the VStack will it make both the subject and title underline?

Reply:
Yep, I set it to the VStack.